tests/format-quick-bar.spec.ts

```diff
@@ -38,17 +42,16 @@ test('should format quick bar be able to format text', async ({ page }) => {
   // drag only the `456` paragraph
   await dragBetweenIndices(page, [1, 0], [1, 3]);
 
-  const boldBtnLocator = page.locator(`.format-quick-bar [data-testid=bold]`);
-  const italicBtnLocator = page.locator(
-    `.format-quick-bar [data-testid=italic]`
-  );
-  const underlineBtnLocator = page.locator(
-    `.format-quick-bar [data-testid=underline]`
-  );
-  const strikeBtnLocator = page.locator(
-    `.format-quick-bar [data-testid=strike]`
+  const formatQuickBarLocator = page.locator(`.format-quick-bar`);
```

Comment:
Guess we can remove the `Locator` naming suffix since it can be inferred by TypeScript